fem/geom.cpp

```diff
@@ -251,6 +251,32 @@ void Geometry::GetRandomPoint(int GeomType, IntegrationPoint &ip)
    }
 }
 
+
+namespace internal
+{
+
+// Fuzzy equality operator with absolute tolerance eps
+inline bool NearlyEqual(double x, double y, double eps)
```

Comment:
I couldn't find equivalent functions in mfem.  If they already exist, we can use them.